django_freeradius/api/views.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib.auth import get_user_model
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+User = get_user_model()
+
+
+@api_view(['POST'])
+def authorize(request):
+    username = request.POST.get('username')
+    password = request.POST.get('password')
```

Comment:
forgot to mention django-rest-framework views `request.data` which you can use in these cases, therefore change to: `request.data.get(...)`